backend/models/user.py

```diff
@@ -211,6 +211,12 @@ def add_institution(self, institution_key):
             self.add_permission("publish_survey", institution_key.urlsafe())
             self.put()
 
+    def add_institution_admin(self, institution_key):
+        """Add a institution admin to user."""
```

Comment:
What? Explain better. It seems that add a User to that list but receives a institituion_key.